bands/templates/bands/index.html

```diff
@@ -160,6 +173,9 @@
             <br/></br>
         </td>
     </tr>
+    {% endif %}
+
+    {% include "bands/bandextra.html" %}
```

Comment:
Django ignores the current directory and instead looks for all templates relative to the list of directories in the ``TEMPLATE_DIRS`` setting:
https://docs.djangoproject.com/en/1.9/ref/settings/#template-dirs

Interestingly, the act of looking up the docs for this setting led me to notice that we're using a now-deprecated way of specifying the template dirs and should probably change that.  I'd make a ticket for it, but I think it falls under the existing "update all of the dependencies in requirements.txt" task that's already ticketed out.